components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -519,7 +528,9 @@ void AdsServiceImpl::ShutdownAndResetState() {
   VLOG(6) << "Resetting ads state";
 
   profile_->GetPrefs()->ClearPrefsWithPrefixSilently("brave.brave_ads");
+  // if (local_state_) {
```

Comment:
?

Reply:
This was to test https://github.com/brave/brave-core/pull/20433#discussion_r1681694541.